tests/components/MCQ.test.ts

```diff
@@ -1,25 +1,53 @@
 import { describe, test, expect } from "vitest";
 import { mount } from "@vue/test-utils";
-import { title } from "@data/question-data.json";
+import { title, options } from "@data/question-data.json";
 import MCQ from "@components/MCQ.vue";
+import { optionMount } from "./MCQUtils";
 
 describe("MCQ.vue", () => {
   test("Renders component", () => {
     const wrapper = mount(MCQ, {
       props: {
         title,
+        options,
       },
     });
     expect(wrapper.exists()).toBe(true);
     expect(wrapper.text()).toContain("MCQ Test");
   });
+
   test("Renders component with title", () => {
     const wrapper = mount(MCQ, {
       props: {
         title,
+        options,
       },
     });
     // Rather than looking at the whole wrapper, targetting specific class.
```

Comment:
Does this comment add meaning, or create noise?

Reply:
Yeah probably creates noise, and maybe the one below it as well.